meta-lmp-base/recipes-core/initrdscripts/initramfs-framework/ostree_factory_reset

```diff
@@ -34,15 +36,34 @@ factory_reset() {
 	if [ "${keep_sota}" = "true" ]; then
 		msg "Keeping current ${OSTREE_DISTRO} SOTA content"
 		FIND_EXCLUDE="${FIND_EXCLUDE} ! -name 'sota'"
+		keep_rpmb="true"
 	fi
 	if [ "${keep_docker}" = "true" ]; then
 		msg "Keeping current docker content"
 		FIND_EXCLUDE="${FIND_EXCLUDE} ! -name 'lib'"
 	fi
 
+	cd ${OSTREE_VAR}
```

Comment:
Not related to this PR, but to help testing this approach you can just call `sh` and iterate over the commands manually.